recipes/draco/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(test_package)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)
```

Comment:
set_target_properties is CMake 3+. Use `set_property` instead.

Reply:
fixed